ios/FluentUI.Demo/FluentUI.Demo/Demos/ContactViewDemoController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class ContactViewDemoController: DemoController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.lightGray
```

Comment:
>lightGray [](start = 39, length = 9)

I don't know if I like this, but I see how it shows that the card is a square. I wonder if we should set the background that one would use in a real life scenario (same color of the card), for better demonstration purpose

Reply:
Are you suggesting to keep the background the same colour as the other demo controllers? This is definitely possible, but I feel like it would be harder for the consumer to understand where the `ContactView` begins and where it ends since the `ContactView` colour is the same as the background from what I understand.

Reply:
i'm on the fence. I think it looks unclean in the demo and it would be better to represent it as the real users will representing, which is, with a matching color. But up to you :)